### ISSUE TITLE
myrasec_settings "available_attributes.extended_stats" issue

### ISSUE THREAD

**Issue by dshumsky** [Myra-Security-GmbH/terraform-provider-myrasec#133]
Hello

After importing state for 2nd-level domain settings:
terraform import 'myrasec_settings.sk["ALL-12345"]' ALL-12345

terraform plan should return empty plan, but it always returns this:
```
resource "myrasec_settings" "domain" {
      ~ available_attributes            = [
          - "extended_stats",
            # (22 unchanged elements hidden)
        ]
        id                              = "ALL-12345"
        # (49 unchanged attributes hidden)
    }
```
make it really unusable

The problem started to appear in the new version (1.63.0)

**Comment by smashi** [comment]
Hi @dshumsky,

thanks for your response, i will check that an inform you when we found a solution.

BR Tim

**Comment by smashi** [comment]
Hi @dshumsky,

we found the reason  and a solution for this, we will provide a new version soon.

BR Tim

**Comment by smashi** [comment]
Hi @dshumsky,

we released version v1.63.1 where this issue is fixed.

BR Tim
